src/instruments/src/ISIS/AirspeedIndicator.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Bug, BugType } from './Bug';
+import { VerticalTape } from './ISISUtils';
+
+type AirspeedIndicatorProps = {
+    maskWidth: number;
+    indicatedAirspeed: number;
+    bugs: Bug[]
+}
+
+type TickProps = { maskWidth: number; offset: number, airspeed: number }
+
+const Tick: React.FC<TickProps> = ({ maskWidth, offset, airspeed }) => {
```

Comment:
I really do not recommend having the offset as a prop. The offset is gonna be the same for all ticks relative to their initial position. This will cause the execution of more code compared to just shifting the group of ticks.

Reply:
I agree. I think I should also get rid of the `maskWidth` being passed around all over the place.

Reply:
I looked into this more and realized that `offset` is just a statically computed value, meaning the offset based on the corresponding airspeed the tick is indicating. So this *is* just the initial position. When the aircraft's airspeed changes, `offset` does not change. The entire svg is translated instead.